app/controllers/kudos_controller.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class KudosController < ApplicationController
```

Comment:
nie powinienes uzywac scaffoldow, bo te zabieraja Twoim palcom potrzebna praktyke. Co prawda, w drugim sprincie jeden task wymaga ich uzycia, ale wylacznie po to, abyscie wiedzieli ze cos takiego istnieje. Prosze, abys juz wiecej ich nie uzywal ✌️